README.md

```diff
@@ -18,18 +12,9 @@
 </details>
   
 ## Installation
-**1.** Firstly, ensure that you are in the Steam Beta. To do this, go to Steam Settings, then select Interface, and choose Steam Beta under the Client Beta Participation section. Without this, you won't be able to use the SpaceTheme.
-
-**2.** Next, download the Millennium Steam Patcher from [GitHub](https://github.com/ShadowMonster99/millennium-steam-patcher). Follow the instructions provided to install the patcher.
-
-**3.** After that, download the SpaceTheme.zip file from [Discord](https://discord.com/channels/1104516050537685144/1104552443221655603) or [GitHub](https://github.com/SpaceEnergy/SpaceTheme-Steam/releases).
-
-**4.** Once you have downloaded the `SpaceTheme.zip` file, go to `C:\Program Files (x86)\Steam\steamui\skins` and paste the .zip file there.
-
-**5.** Next, extract the .zip file, and you will see a folder named `Steam SpaceTheme`. Delete the .zip file from the skins folder.
-
-**6.** Go back to Steam Settings, select Interface, and choose the SpaceTheme from the dropdown menu.
-
-**7.** Finally, restart Steam, and you should have the SpaceTheme installed. Enjoy your new skin!
-
+1. Download the [Millennium](https://steambrew.app/). Follow the instructions provided to install the patcher.
+1. Open Steam and navigate to `Steam -> Settings -> Interface -> Open Millennium`
+1. With the Millennium interface open, drag the install button overtop of the interface. 
+1. From there, you will see a install prompt. Now simply drop the theme
+1. Select the theme from the interface after it installs.
```

Comment:
the listing numbers are not correct and maybe add a text at step 2 that you are not allowed to use Steam Beta
```suggestion
1. Download the [Millennium](https://steambrew.app/). Follow the instructions provided to install the patcher.
2. Open Steam and navigate to `Steam -> Settings -> Interface -> Open Millennium` _(Make sure you're not in the Steam beta)_
3. With the Millennium interface open, drag the install button overtop of the interface. 
4. From there, you will see a install prompt. Now simply drop the theme
5. Select the theme from the interface after it installs.
```

Reply:
using 1. is the proper markdown syntax. it auto increments for you. As for the Steam Beta notice, it should be fine considering there is already a warning on the front page of the website. If we update it now, we will just have to update it again later!